mockserver_friendly/__init__.py

```diff
@@ -51,9 +51,10 @@ def request(method=None, path=None, querystring=None, body=None, headers=None, c
     )
 
 
-def response(code=None, body=None, headers=None, cookies=None, delay=None):
+def response(code=None, reason=None, body=None, headers=None, cookies=None, delay=None):
```

Comment:
Hi, Thanks for the patch !
Can you move the reason as the last parameter. It will prevent us breaking someone using Response as a mock not having specified the kwarg, eg 

    mock.should_return = response(200, 'my body') 

such a unit test would be broken by this change, adding as last params solve it.

Reply:
Done